docs/openapi-qtsp.yaml

```
@@ -0,0 +1,185 @@
+openapi: 3.0.3
+
+info:
+  title: Firma con IO QTSP
+  description: Any APIs called by IO Backend to QTSP
+  version: 1.0.0
+
+paths:
+
+  /tos:
+    summary: Request terms of service
+    get:
+      operationId: GetTos
+      tags:
+        - QTSP
+
+      responses:
+        '200':
+          description: Terms of service to be accepted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TOS"
+        
+        default:
+          description: Generic error
+      
+      
+      
+  /login:
+    summary: Request login to QTSP
+    post:
+      operationId: GetChallenge
+      tags:
+        - QTSP
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Auth'
```

```suggestion
              $ref: '#/components/schemas/GetChallengeInput'
```

I'd put some reference to "challenge" here since we're not authenticating.